src/section_photo/section_photo.css

```
@@ -9,6 +9,28 @@
     position: relative;
 }
 
+.pagination_data {
+    display: flex;
+    gap: 20px;
+}
+
+.pagenation {
```

```suggestion
.pagination {
```
just fix it everywhere really